index.html

```diff
@@ -132,15 +132,37 @@ <h3 class="worktitle">My Recent Work</h3>
                     <li><img src="./pictures/Rectangle70.png" alt=""></li>
                     <li>Ruby</li>
                 </ul>
```

Comment:
- [ ] As the previous reviewer has suggested, kindly remove all the hard-coded data you have in your index.html file since you are already generating it in the js file. 👍